bin/commit_xpi.sh

```diff
@@ -8,39 +8,23 @@
 
 set -e
 
-# TODO: Make env variable in .taskcluster.yml that gets passed to this script to specify
-# which channel(s) to test in
-
-# Repo URLs:
-#   - Nightly: https://hg.mozilla.org/mozilla-central
-#   - Beta: https://hg.mozilla.org/releases/mozilla-beta
-#   - Release: https://hg.mozilla.org/releases/mozilla-release
-echo ">>> clone Firefox"
-# TODO: Investigate how to get a shallow clone of Firefox -- this full clone takes 3ish minutes alone.
+# Note: If clone fails with error: `abort: unexpected response from remote server: empty string.`,
+# that clonebundle may be corrupted. Try another URL to confirm and talk to sheehan in #vcs in IRC.
```

Comment:
Not sure we should call out sheehan by name... mentioning the irc channel is ok.